src/profile-logic/call-tree.js

```diff
@@ -259,8 +252,13 @@ export class CallTree {
         icon = ExtensionIcon;
       }
 
+      const formatNumber = this._isIntegerInterval
+        ? _formatIntegerNumber
+        : _formatDecimalNumber;
+
       displayData = {
-        ...this.getTimingDisplayData(callNodeIndex),
+        totalTime: `${formatNumber(totalTime)}`,
+        selfTime: selfTime === 0 ? '—' : `${formatNumber(selfTime)}`,
```

Comment:
Small nit: We don't need to use a template string here as the the only content is the result of `formatNumber`. This could be useful if the result of the function wasn't a string, but here it's a string. So you can simply use: `totalTime: formatNumber(totalTime)` and the same for `selfTime`.